### ISSUE TITLE
Managed to deplete a infinite Ironium vein with drone miners.

### ISSUE THREAD

**Issue by Chrisinnater5** [TheKilltech/rift_balance#3]
^^^
At least i didn't let them touch the Titanium one. I was also testing out using 8 lvl2 stations at the same time so I probably shouldn't be surprised. xD

**Comment by TheKilltech** [comment]
Interesting catch. Okay, the underground deposits weren't supposed to be mine-able with the drone miner anyway. Other then that, this might be an issue with the base game, albeit i don't remember seeing infinite deposits in the base game, except for geothermal and flammable gas vents, which cannot be mined. So it's indeed a situation which occurs only with the mod. will investigate this at some point, but it's no priority.

**Comment by Chrisinnater5** [comment]
Yeah, fair enough, I'm just more careful with where I place them. I probably should have mentioned I am running other mods, but nothing that should modify deposits/cheaty/etc. Also, forgive my lack of experience on giphub, but if you'd like me to test anything specifically let me know ^_^

**Comment by ameno-gdrive** [comment]
On mod version 0.3.12  I can't seem to get the drone miners to do anything.  I placed them within range of deposit outcrops like carbonium / ironium but the drone miner building does not deploy any drones to mine anything.

**Comment by TheKilltech** [comment]
thanks. i do remember that it worked for some time. but i didn't do much with it. should probably make it work with the ore mill. but as it is similar to the cultivators, i am not sure this will work. anyway, i did intend to look into it, just didn't to it yet.

still looking for a good use case for it in the game. maybe its core advantage would be the reduced need for piping as it can cover multiple dispersed deposits, though it would have lower yield overall. 

regarding the originally reported issue, i had the suspicion, that because for resource deposits the number of resources is an independent property from whether it is infinite, the drone miners could potentially ignore it. however, the gui won't show the actual number in the deposit, so it would be difficult to check and would be a bug in vanilla as well. intended to check it by creating small but infinite deposits and check if they vanish after being mined. repeat with the same with large infinite deposits - they wouldn't vanish in a time limited test.

**Comment by TheKilltech** [comment]
<img width="2555" height="1439" alt="Image" src="https://github.com/user-attachments/assets/33f334b6-31df-42db-a52e-b2fec7cda023" />

the drone miner still works for me. drones spawn on it and move around to nearby deposits. it harvests both surface and underground deposits - not even sure how it is determining what i can mine. i wonder if i should keep it like that or not. seems it to have quite low yield overall - though this will be an issue with how far the drones travel as they try to harvest all deposits nearby.

i noticed that in some of your screenshots there are buildings not from this mod, so there is a potential chance that some other mods breaks the drones for you. technically, i am also using a different version since i am always playing using the dev branch version. that may also be an issue
